src/Features/ERDDAP/Platform/Observations/Table/item.tsx

```diff
@@ -1,4 +1,7 @@
-import React, { useState } from "react"
+/**
+ * A single row in the current or all conditions tables
+ */
+import * as React from "react"
```

Comment:
Codacy found an issue: ['React' was used before it was defined.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6166252)